### PR TITLE
Tokens

### PR DESCRIPTION
**Pull request by mirii1994**
Upgrade to client v1.6.0
Add log shipping token endpoint

### CODE REVIEW

File: docs/data-sources/log_shipping_token.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Log Shipping Token Provider
+
+Provides a Logz.io log shipping token resource. This can be used to create and manage Logz.io log shipping tokens.
+
+* Learn more about log alerts in the [Logz.io Docs](https://docs.logz.io/api/#tag/Manage-log-shipping-tokens)
```
Comment (Doron-Bargo) [inline docs/data-sources/log_shipping_token.md:5]: please replace the "log alerts" with log token or something relevant

File: docs/resources/log_shipping_token.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Log Shipping Token Provider
+
+Provides a Logz.io log shipping token resource. This can be used to create and manage Logz.io log shipping tokens.
+
+* Learn more about log alerts in the [Logz.io Docs](https://docs.logz.io/api/#tag/Manage-log-shipping-tokens)
```
Comment (Doron-Bargo) [inline docs/resources/log_shipping_token.md:5]: please replace "log alerts" with log shipping token or something relevant

File: logzio/resource_log_shipping_token.go [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+// Creates a new log shipping token in logz.io
+func resourceLogShippingTokenCreate(d *schema.ResourceData, m interface{}) error {
+	createToken := log_shipping_tokens.CreateLogShippingToken{Name: d.Get(logShippingTokenName).(string)}
+	tokenLimits, err := logShippingTokenClient(m).GetLogShippingLimitsToken()
+	if err != nil {
+		return err
+	}
+
+	// Check if we exceeded the number of max allowed tokens
+	if tokenLimits.MaxAllowedTokens > tokenLimits.NumOfEnabledTokens {
```
Comment (Doron-Bargo) [inline logzio/resource_log_shipping_token.go:88]: can you replace the order?
tokenLimits.NumOfEnabledTokens <= tokenLimits.MaxAllowedTokens 
Its more readable that way

File: logzio/resource_log_shipping_token_test.go [hunk 4]
```diff
@@ -0,0 +1,77 @@
+package logzio
```
Comment (Doron-Bargo) [inline logzio/resource_log_shipping_token_test.go:1]: please add negative tests

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
Great work, please look at my comments

File: logzio/resource_log_shipping_token.go (hunk 3 above)
Comment (mirii1994) [reply logzio/resource_log_shipping_token.go:88]: I think it should be < rather than <=. if it equal they can't create more.

Comment (Doron-Bargo) [reply logzio/resource_log_shipping_token.go]: wasn't sure but if you say so i believe you
anyway my comment was on the order to make it more readable

File: docs/data-sources/log_shipping_token.md (hunk 1 above)
Comment (mirii1994) [reply docs/data-sources/log_shipping_token.md]: My bad. fixed.

File: docs/resources/log_shipping_token.md (hunk 2 above)
Comment (mirii1994) [reply docs/resources/log_shipping_token.md]: My bad. fixed.

File: logzio/resource_log_shipping_token_test.go (hunk 4 above)
Comment (mirii1994) [reply logzio/resource_log_shipping_token_test.go:1]: Added

**Review by Doron-Bargo — APPROVED** [review]
Good work

**Pull request merged**